magenta-lib/src/main/scala/magenta/deployment_type/Lambda.scala

```diff
@@ -16,6 +16,9 @@ object Lambda extends DeploymentType  {
       |
       |It is recommended to use the `bucket` parameter as storing the function code in S3 works much better when using
       |cloudformation.
+      |
+      |You will need to add this as a dependency to your autoscaling deploy in your riff-raff.yaml to guard against race conditions.
```

Comment:
I'm unclear about how a `Lambda` deploy could cause problems for an `autoscaling` deploy. Do you have an example?

Reply:
Ah - lambda can conflict with cloudformation in the same way as autoscaling can conflict with cloudformation. lambda shouldn't be able to conflict with autoscaling.

Reply:
Thanks! Will fix the typo